### ISSUE TITLE
TypeError: checkSizzleExists(...).then(...).then(...).thenCatch is not a function

### ISSUE THREAD

**Issue by hellboy81** [goodeggs/webdriver-sizzle#8]
```js
expect('#bcomments').dom.to.be.visible();
```

```
TypeError: checkSizzleExists(...).then(...).then(...).thenCatch is not a function
    at finder (..\node_modules\webdriver-sizzle\lib\webdriver_sizzle.js:47:10)
    at call (...\selenium-webdriver\lib\webdriver.js:1037:28)
    at ...node_modules\selenium-webdriver\lib\webdriver.js:916:19
    at ManagedPromise.invokeCallback_ (node_modules\selenium-webdriver\lib\promise.js:1384:14)
    at TaskQueue.execute_ (\node_modules\selenium-webdriver\lib\promise.js:3092:14)
    at TaskQueue.executeNext_ (node_modules\selenium-webdriver\lib\promise.js:3075:27)
    at asyncRun (node_modules\selenium-webdriver\lib\promise.js:2982:25)
    at node_modules\selenium-webdriver\lib\promise.js:676:7
```

**Comment by dwightdegroff** [comment]
Anyone find a workaround for this? Anytime I try to pass callback or chain with a promise, I get this error.

**Comment by floppyTop** [comment]
I'm receiving the same error, with the following test code:

```describe('Homepage Tests', function() {
    before(function() {
        driver.get(baseURL);
        console.log('Tests mounted, driver started');
    });

    after(function() {
        driver.quit();
    });
    
    it('should contain an h1, containing "We"', () => {
        return expect('h1 span')
            .dom.to.be.visible();
        done();
    });
});
`

**Comment by twpol** [comment]
The around I've come up with is this:

``

**Comment by twpol** [comment]
It looks like the fix is to replace `thenCatch` with `catch` in this library, so the workaround I've come up with is:

```javascript
// Hack to fix https://github.com/goodeggs/webdriver-sizzle/issues/8:
const seleniumWebDriverPromise = require('selenium-webdriver/lib/promise');
seleniumWebDriverPromise.Promise.prototype.thenCatch = seleniumWebDriverPromise.Promise.prototype.catch;
```

**Comment by serhalp** [comment]
This appears to have been fixed a couple years ago by #10.
